src/coverup/coverup.py

```diff
@@ -55,7 +55,12 @@ def Path_dir(value):
     ap.add_argument('--model', type=str,
                     help='OpenAI model to use')
 
-    ap.add_argument('--model-temperature', type=str, default=0,
+    ap.add_argument('--prompt-family', type=str,
+                    choices = ['gpt', 'claude'],
+                    default = 'gpt',
+                    help='Prompt style to use')
+
+    ap.add_argument('--model-temperature', type=float, default=0,
```

Comment:
This line fixes a bug with temperature, which was being parsed as a string instead of a float.